content/docs/hooks-custom.md

```diff
@@ -6,14 +6,14 @@ next: hooks-reference.html
 prev: hooks-rules.html
 ---
 
-*Hooks* are an upcoming feature that lets you use state and other React features without writing a class. They're currently in React v16.8.0-alpha.1.
+*Hooks* let you use state and other React features without writing a class.
```

Comment:
Do you think "are a new feature that lets" would be better? I ask mainly because it makes it more obvious why the Hooks docs are in their own section and not properly incorporated through the entire docs.

Reply:
I was a bit annoyed by "new feature that lets you use [...] React features" but I agree with the point. Any other suggestions?

Reply:
Maybe: “Hooks are a new addition in React 16.8. They let”